dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -711,49 +709,7 @@ public void handleLabels(String entryPath, Set<String> addsSet, Set<String> remo
 
     @Override
     protected void handleVerifyUnverify(String entry, String versionName, String verifySource, boolean unverifyRequest, boolean isScript) {
-        boolean toOverwrite = true;
-
-        try {
-            DockstoreTool tool = containersApi.getContainerByToolPath(entry, null);
-            List<Tag> tags = Optional.ofNullable(tool.getWorkflowVersions()).orElse(new ArrayList<>());
-            final Optional<Tag> first = tags.stream().filter((Tag u) -> u.getName().equals(versionName)).findFirst();
-
-            if (first.isEmpty()) {
-                errorMessage(versionName + " is not a valid tag for " + entry, Client.CLIENT_ERROR);
-            }
-            Tag tagToUpdate = first.get();
-
-            VerifyRequest verifyRequest = new VerifyRequest();
-            if (unverifyRequest) {
-                verifyRequest = SwaggerUtility.createVerifyRequest(false, null);
-            } else {
-                // Check if already has been verified
-                if (tagToUpdate.isVerified() && !isScript) {
```

Comment:
Leave this part as commented code, I kinda like the structure of how it checks whether something is verified and asks to overwrite and want the implementer to get that hint.